### PR TITLE
[fix] save our log files to the user's home directory

### PR DESCRIPTION
**Pull request by Hiro-Nakamura**
This is an attempt to relocate the log files to the User's home directory rather than to the base directory of the BibleKaraoke app.

### CODE REVIEW

File: public/cli/lib/commands/convert.js [hunk 1]
```diff
@@ -195,7 +196,20 @@ Command.run = function(options) {
...
     // make sure the logging directory exists
-    var pathToLogDir = path.join(__dirname, "..", "..", "..", "..", "logs");
+    var pathToLogDir;
+    switch(process.platform) {
+        case "darwin":
+            pathToLogDir = path.join(os.homedir(), "Library", "Logs", "bbk", "logs");
+            break;
+
+        case "win32":
+            pathToLogDir = path.join(os.homedir(), "AppData", "Roaming", "bbk", "logs");
```
Comment (irahopkinson) [inline public/cli/lib/commands/convert.js:206]: Change `bbk` folder to `bible-karaoke`. In Windows (at least) there is already a folder for it when the app is installed.

File: public/cli/lib/commands/convert.js [hunk 2]
```diff
@@ -195,7 +196,20 @@ Command.run = function(options) {
  function prepareLogger( done, numLogsToKeep=10 ) {
 
     // make sure the logging directory exists
-    var pathToLogDir = path.join(__dirname, "..", "..", "..", "..", "logs");
+    var pathToLogDir;
+    switch(process.platform) {
+        case "darwin":
+            pathToLogDir = path.join(os.homedir(), "Library", "Logs", "bbk", "logs");
```
Comment (irahopkinson) [inline public/cli/lib/commands/convert.js:202]: Th MacOS doesn't need the final `logs` folder according to Eric's info.

Comment (Hiro-Nakamura) [reply public/cli/lib/commands/convert.js:202]: yeah, but for consistency sake, do you want to take ```"logs"``` out of MacOS, but leave it in for the rest?

Comment (irahopkinson) [reply public/cli/lib/commands/convert.js:202]: Yes, as it is already in a specific `Logs` folder. On Mac it is ordered by logs first then app; on Linux and Windows it is ordered by app first then logs.

**Comment by Hiro-Nakamura** [comment]
Updated.

Also, do you want this merged back into 0.3.3 tag?

**Comment by irahopkinson** [comment]
I've rebased my release branch now that your fix is on `master`.

**Pull request merged**